x-pack/plugins/security_solution/common/api/detection_engine/alert_assignees/set_alert_assignees_route.schema.yaml

```diff
@@ -4,12 +4,15 @@ info:
   version: '2023-10-31'
 paths:
   /api/detection_engine/signals/assignees:
-    summary: Assigns users to alerts
     post:
       x-labels: [ess, serverless]
       x-codegen-enabled: true
       operationId: SetAlertAssignees
-      description: Assigns users to alerts.
+      summary: Assign and unassign users from detection alerts
+      description: |
+        Assign users to detection alerts, and unassign them from alerts.
+        > info
+        > You cannot add and remove the same assignee.
```

Comment:
A little confusing to me

Reply:
@szwarckonrad This is meant to convey that you can't both add and remove the same user as an assignee in the same request. We have the same note in the [equivalent ESS API docs](https://www.elastic.co/guide/en/security/current/signals-api-overview.html#_request_body_11).

Do you think adding "in the same request" would make the description clearer, or do you have another suggestion?

Reply:
> Do you think adding "in the same request" would make the description clearer, or do you have another suggestion?

looks like a good idea